src/components/views/avatars/MemberStatusMessageAvatar.js

```diff
@@ -40,77 +40,91 @@ export default class MemberStatusMessageAvatar extends React.Component {
 
     constructor(props, context) {
         super(props, context);
+
+        this.state = {
+            hasStatus: this.hasStatus,
+        };
     }
 
     componentWillMount() {
         if (this.props.member.userId !== MatrixClientPeg.get().getUserId()) {
             throw new Error("Cannot use MemberStatusMessageAvatar on anyone but the logged in user");
         }
+        if (!SettingsStore.isFeatureEnabled("feature_custom_status")) {
+            return;
+        }
+        const { user } = this.props.member;
+        if (!user) {
```

Comment:
one of these days we should track down why `user` is sometimes null :(

Reply:
Ah, I didn't realize it was so mysterious... 😭 (Added to my list of things to investigate in the future.)